spec/index.html

```diff
@@ -1331,7 +1331,7 @@ <h3>RDFS entailment</h3>
 
     <p>where aaa is an IRI, are true in all RDFS interpretations.</p>
 
-    <section id="rdfs_patterns" class="informative">
+    <section id="rdfs_patterns" class="informative" data-dfn-for="RDFS entailment patterns">
```

Comment:
The data-dfn-for would seem to be intended to add the value as a prefix to the rule definition anchors, but it doesn’t seem to actually changed anything.

Reply:
I can't really investigate right now, but a question on spec-prod@w3.org describing what you're trying to do might get some helpful feedback. I've always used data-dfn-for in conjunction with WebIDL definitions, for instance in JSON-LD API.

Reply:
It is supposed to work, according to Tab Atkins.